appsec-kit-v24/pom.xml

```diff
@@ -14,7 +14,7 @@
     <name>AppSec Kit Vaadin 24</name>
 
     <properties>
-        <vaadin24.version>24.1.5</vaadin24.version>
+        <vaadin24.version>24.1.9</vaadin24.version>
```

Comment:
Better use Flow version here, not platform. I'd rename the property `flow.version` (then we can use profiles to set different versions for CI/CD).